examples/app_commands/slash_basic.py

```diff
@@ -0,0 +1,31 @@
+import discord
+
+bot = discord.Bot()
+
+# If you use commands.Bot, @bot.slash_command should be used for
+# slash commands. You can use @bot.slash_command with discord.Bot aswell

```

Comment:
This seems like we're over complicating things. Why not leave the `command` decorator for the commands extension, and `slash_command` for other commands?

Reply:
`command` is just a shortcut for `application_command`. It is only for the discord.Bot class

Reply:
I see, we'd blow this away when using `discord.ext.commands.Bot` 👍